services/syncProjectsRequiredForListing.ts

```diff
@@ -2,31 +2,33 @@ import { Project } from '../entities/project';
 import { schedule } from 'node-cron';
 import { getRepository } from 'typeorm';
 
-// @ts-ignore
 import config from '../config';
 
 const cronJobTime =
-  String(config.get('VERIFY_DONATION_CRONJOB_EXPRESSION')) || '0 0 * * *';
+  (config.get('MAKE_UNREVIEWED_PROJECT_LISTED_CRONJOB_EXPRESSION') as string) ||
```

Comment:
I thinks it's better to have different condif for different cron jobs

Reply:
Whoops I thought I changed it. Thanks